src/main/java/org/nastya/controller/UserController.java

```diff
@@ -35,10 +36,10 @@ public ResponseEntity<UserDTO> getUserByUsername(@PathVariable String username)
     }
 
     @PostMapping
-    public ResponseEntity<UserDTO> createUser(@RequestBody UserDTO userDTO) {
+    public ResponseEntity<UserDTO> createUser(@RequestBody UserDTO userDTO, @RequestHeader(HttpHeaders.AUTHORIZATION) HttpHeaders headers) {
```

Comment:
Why did you add the "HttpHeaders headers" here? It is not a part of your task. Remove it.